src/CommonLib/utility.cxx

```diff
@@ -627,3 +627,44 @@ UTILITY::LogEvent(
         fwprintf(stderr, L"ERROR: %s\n", pstrMsg);
     }
 }
+
+HRESULT
+UTILITY::GetExpandedEnvironmentString(
```

Comment:
you don't need this function. STRU has a method called CopyAndExpandEnvironmentStrings. you can use it directly